src/test/resources/capabilities.yml

```diff
@@ -0,0 +1,58 @@
+testEndpoint: https://bstackdemo.com
+
+driverType: localDriver
+
+localDriver:
+  platforms:
+    - name: chrome
+      driverPath: /Users/bhagyesh/Downloads/Driver/chromedriver
+      capabilities:
+      #    -
+#    - name: safari
+#      driverPath:
+#      capabilities:
+#      #
+
+remoteDriver:
```

Comment:
remoteDriver shouldn't be there for this profile?

Reply:
Removed Remote Driver from capabilities.yml